code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -78,6 +79,24 @@
 		"is twisting their own neck!",
 		"is holding their breath!")
 
+	species_abilities = list(
+		/mob/living/carbon/human/proc/tail_lash
+		)
+
+/mob/living/carbon/human/proc/tail_lash()
+	set category = "IC"
+	set name = "Tail Lash"
+	set desc = "lash someone with your tail"
+
+	for(var/mob/living/carbon/human/C in orange(1))
+		var/obj/item/organ/external/E = C.get_organ(pick("l_leg", "r_leg", "l_foot", "r_foot", "groin"))
+		if(E)
+			visible_message("<span class='danger'> [src] smacks [C] in the [E] with thier tail!</span>", "<span class='danger'>You hit [C] with your tail!</span>")
```

Comment:
Might even be able to add a little animation here to go along with the sound effect where the player 'spins' (not flips) to indicate the lashing (should they not be restrained).